### PR TITLE
Fix fellowship

### PR DESCRIPTION
**Pull request by TLESORT**
There were an errors "__init__() got multiple values for argument in PytorchDataset class. It is fixed. I also added some slow tests for Fellowships

### CODE REVIEW

File: docs/_tutorials/scenarios/scenarios.rst [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+    # Then the dataset is provided to a continuum class that will process it to create the sequence of tasks
+    # the continuum might get specific argument, such as number of tasks.
+    continuum = MyContinuumClass(
+        continual_dataset, SomeOtherArgumetns
+    )
+
+    # The continuum can then be enumerate tasks
+    for task_id, dataset in enumerate(continuum):
+          # train dataset is a normal data loader like in pytorch that can be used to load the task data
```
[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:26]: Does it mean that it should be possible to do:
```
import torch.utils.data.DataLoader as DataLoader

loader=DataLoader(dataset, batch_size=64, shuffle=False, sampler=None,
           batch_sampler=None, num_workers=0, collate_fn=None,
           pin_memory=False, drop_last=False, timeout=0,
           worker_init_fn=None)

[....]
```

File: docs/_tutorials/scenarios/scenarios.rst [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+
+.. code-block:: python
+	
+    # First we get a dataset that will be used to compose tasks and the continuum
+    continual_daset = MyContinualDataset()
+
+    # Then the dataset is provided to a continuum class that will process it to create the sequence of tasks
+    # the continuum might get specific argument, such as number of tasks.
+    continuum = MyContinuumClass(
```
[the remark this answers is not in the document]
Comment (arthurdouillard) [reply docs/_tutorials/scenarios/scenarios.rst:20]: Kinda yeah, but we put the scenarios in the scenario folder :)

File: docs/_tutorials/scenarios/scenarios.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (arthurdouillard) [reply docs/_tutorials/scenarios/scenarios.rst:26]: Yes, see first example of readme

[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:26]: :fire:

File: docs/_tutorials/scenarios/scenarios.rst [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+        continual_dataset, SomeOtherArgumetns
+    )
+
+    # The continuum can then be enumerate tasks
+    for task_id, dataset in enumerate(continuum):
+          # train dataset is a normal data loader like in pytorch that can be used to load the task data
+          for x, y, t in dataset:
```
[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:27]: yes

File: docs/_tutorials/scenarios/scenarios.rst [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+   continuum = ClassIncremental(dataset, increment=2)
+
+    # The continuum can then be enumerate tasks
+    for task_id, dataset in enumerate(continuum):
+
+        # We use here a cool function to split the dataset into train/val with 90% train
+        train_dataset, val_dataset = split_train_val(dataset, split_val = 0.1)
+
+         # train dataset is a normal data loader like in pytorch that can be used to load the task data
+         for x, y, t in train_dataset:
```
[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:49]: yes

File: docs/_tutorials/scenarios/scenarios.rst [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+A practical example with split MNIST:
+
+.. code-block:: python
+
+   from continuum.datasets import MNIST
+   from continuum.tasks import Taskset, split_train_val
+   dataset=MNIST("my/data/path", download=True, train=True)
+
+   # split MNIST with 2 classes per tasks -> 5 tasks
+   continuum = ClassIncremental(dataset, increment=2)
```
[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:40]: bien vu

File: docs/_tutorials/scenarios/scenarios.rst [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+
+
+- Permutations Incremental [source](https://github.com/Continvvm/continuum/blob/master/continuum/scenarios/permutations.py)
+is a famous case of TransformationIncremental class, in this case the transformation is a fixed pixel permutation. Each task has a specific permutation.
+The scenarios is then to learn a same task in various permutation spaces.
+
+.. code-block:: python
+
+    from continuum.datasets import MNIST
+    from continuum.scenarios import Permutations
```
[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:178]: I will test before the PR of the docs branch

File: docs/_tutorials/scenarios/scenarios.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:20]: I see what you mean

File: docs/_tutorials/scenarios/scenarios.rst [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+
+         # train dataset is a normal data loader like in pytorch that can be used to load the task data
+         for x, y, t in train_dataset:
+                # data, label, task index
+                # train on the task here
+
+    # For testing we need to create another loader (It is importan to keep test a train separate)
+    dataset_test=MNIST("my/data/path", download=True, train=False)
+
+    # Choice 1: Test can be used as simple dataset
```
[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst:56]:
```
dataset_test = MNIST("my/data/path", download=True, train=False)
continuum = ClassIncremental(dataset_test, increment=2)
datasets = continuum[:]
test_loader = DataLoader(datasets)
```
C'est ça par exemple?

Si on veux avoir juste toute les données, c'est peut être un peu laborieux....

[the remark this answers is not in the document]
Comment (TLESORT) [reply docs/_tutorials/scenarios/scenarios.rst]: We can keep it like this ...

**Pull request merged**